etc/lmod-setup.csh

```diff
@@ -0,0 +1,26 @@
+#!/bin/csh
+
+if ( $# == 0 ) then
+   cat << EOF_USAGE
+Usage: source etc/lmod-setup.csh PLATFORM
+
+OPTIONS:
+   PLATFORM - name of machine you are building on
+      (e.g. cheyenne | hera | jet | orion | wcoss2 )
```

Comment:
I would remove the reference to Cheyenne.

Reply:
Done